get-started/index.md

```diff
@@ -151,35 +150,34 @@ $ kubectl apply -f cryostat.yaml
 ```
 
 ### [Open the Cryostat Web UI](#open-the-cryostat-web-ui)
-Let's visit the Cryostat web dashboard UI.
+Let's visit the **Cryostat** web dashboard UI.
 
-We can get there from the `Cryostat` `CR`'s status fields:
+We can get there from the **Cryostat** `CR`'s status fields:
 
 {% include howto_step.html
   details-attributes="open"
   summary="Cryostat Resource Status"
   image-name="cryostat-resource-status.png"
 %}
 
-Or, we can open the application link from the OpenShift Console Topology view:
+Or, we can open the application link from the **OpenShift** Console *Topology* view:
 
 {% include howto_step.html
   details-attributes="open"
   summary="Topology View"
   image-name="topology-view.png"
 %}
 
-We can also find the URL using `oc`:
+We can also find the **URL** using `oc`:
 ```bash
 $ oc get cryostat -o jsonpath='{$.items[0].status.applicationUrl}'
 ```
 
 #### [Authenticate through Cryostat](#authenticate-through-cryostat)
 
 ##### [OpenShift Authentication](#openshift-authentication)
-When deployed in OpenShift, Cryostat will use the existing internal cluster
-authentication system to ensure all requests come from users with correct
-access to the Cryostat instance and the namespace that it is deployed within.
+When deployed in **OpenShift**, **Cryostat** will use the existing **internal cluster** authentication system to ensure all requests come from users with correct
```

Comment:
It only matters for us who are editing the file, but it would be nice to keep the line lengths consistent when just applying new formatting here.